packages/dtd-catalog-exporter/package.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "dtd-catalog-exporter-job",
```

To adhere to the standard of other `package.json` files

```suggestion
  "name": "pagopa-interop-dtd-catalog-exporter-job",
```